Src/Fido2.BlazorWebAssembly/Fido2.BlazorWebAssembly.csproj

```diff
@@ -1,10 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk.Razor">
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
     <TargetFramework>$(SupportedTargetFrameworks)</TargetFramework>
```

Comment:
I'd like one more s here to also make this *`TargetFrameworks`

Reply:
FYI: After .NET8.0 is released next month, I plan on adding .NET8.0 to the TargetFramework list. Having everything managed in directory.props should make this super simple.